.github/workflows/docker.yml

```diff
@@ -1,26 +1,42 @@
 name: Docker
 
 on:
-  pull_request:
```

Comment:
Any reason why `pull_request` trigger is dropped here?

Reply:
As it will run on each push to build the image over and over on each branch (just 1 task per branch, due to concurrency limits), I just left the "push" action only

Reply:
That makes sense to me.